[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 opened the pull request:
http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin
- create cubrid, cubrid UI plugin based on generic plugin
- register new added plugins

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic plugin

longhaseng52 committed:
Merge pull request #2 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- Refactoring file name

longhaseng52 committed:
Merge pull request #3 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic
git commit -F- <<EOF
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic
- refactoring cubrid ui plugin

longhaseng52 committed:
Merge pull request #4 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 committed:
Revert "[TOOLS-4458] Create a Cubrid plugin based on generic"

longhaseng52 committed:
Merge pull request #5 from longhaseng52/revert-4-TOOLS-4458

Revert "[TOOLS-4458] Create a Cubrid plugin based on generic"

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- register new added plugin

longhaseng52 committed:
Merge branch 'devel' into TOOLS-4458

longhaseng52 committed:
Merge pull request #6 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

- create cubrid and cubrid UI plugin based on generic plugin
- register new added plugins

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin
- create cubrid, cubrid UI plugin based on generic plugin
- register new added plugins

longhaseng52 committed:
Merge branch 'devel' into TOOL-4458

longhaseng52 committed:
Merge pull request #7 from longhaseng52/TOOL-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin
- create cubrid, cubrid UI plugin based on generic plugin
- register new added plugins

longhaseng52 commented:
Dear @kisoo-han,

There were some issues with many commits and messages in the previous pull request. I just organized it to make it look clean, that's why I have created a new above pull request here.

kisoo-han commented:
@longhaseng52 ,
I see!

longhaseng52 committed:
Merge branch 'devel' into TOOLS-4458

hwany7seo commented on features/org.jkiss.dbeaver.db.feature/feature.xml:
it was duplicated.

hwany7seo commented on features/org.jkiss.dbeaver.db.ui.feature/feature.xml:
it was duplicated

hwany7seo commented on product/debug/DBeaver.product.launch:
Please add cubrid plugin for debug

hwany7seo commented on plugins/org.jkiss.dbeaver.ext.generic/plugin.xml:
In my opinion, It would be better to move the setting for CUBRID to the CUBRID plugin. 
And the maven repository is not added.

longhaseng52 replied:
Add maven repository to Cubrid driver in Cubrid plugin will be include in ticket (TOOLS-4447).

hwany7seo reviewed (approved):
it's good!
thank you for your effort!

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- fixed duplicated Cubrid in feature.xml
- fixed duplicated Cubrid UI in feature.xml
- added Cubrid plugin for debug
- reverted Cubrid driver in generic plugin to original
